src/components/servicesView/Section.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Typography, Box, Button, Grid } from "@material-ui/core";
+
+const Section = ({ header, description, image, buttons }) => {
+  const buttonList = buttons.map((button) => (
+    <Button
+      data-cy={`button_${button.id}`}
+      variant="contained"
+      color="secondary"
+      href={button.link}
+    >
+      <Typography variant="button">{button.text}</Typography>
+    </Button>
+  ));
+
+  return (
+    <Grid
+      style={styles.section}
+      container
+      alignItems="center"
+      direction="row"
+      data-cy="service-section"
+    >
+      <Grid item xs={12} lg={6} data-cy="image" style={styles.itemContainer}>
+        <img style={styles.image} src={image.url} alt={image.alt} />
+      </Grid>
+      <Grid item xs={12} lg={6} style={styles.itemContainer}>
+        <Typography data-cy="header" variant="h3" component="h3" gutterBottom>
+          {header}
+        </Typography>
+        <Typography
+          data-cy="description"
+          variant="body1"
+          component="p"
+          gutterBottom
+        >
+          {description}
+        </Typography>
+        <Box style={styles.buttonContainer}>{buttons && buttonList}</Box>
+      </Grid>
+    </Grid>
+  );
+};
+
+export default Section;
+
+const styles = {
```

Comment:
should we pull the css to a separate file but still within the same directory?

Reply:
Since we use Material UI we use inline style to overwrite some attributes of some components. So i believe leaving it the best way at the moment. 
But correct me if i'm wrong :)